aws/table_aws_iam_policy.go

```diff
@@ -109,6 +109,13 @@ func tableAwsIamPolicy(_ context.Context) *plugin.Table {
 				Hydrate:     getPolicyVersion,
 				Transform:   transform.FromField("PolicyVersion.Document").Transform(unescape).Transform(policyToCanonical),
 			},
+			{
+				Name:        "policy_usage",
+				Description: "Lists all IAM users, groups, and roles that the specified managed policy is attached to.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:	 getPolicyUsage,
```

Comment:
Please update the format. you can run `gofmt -w filename.go`

Reply:
Sorry about that. I applied gofmt command.